pyproject.toml

```diff
@@ -51,7 +51,8 @@ imaging = [
     'gcsfs',
     'xarray-multiscale',
     'parameterized',
-    'zarr==2.13.3'
+    'zarr==2.13.3',
+    'tqdm==4.64.1'
```

Comment:
just curious, is there a bug with the latest `tqdm`? Locking versions makes perfect sense to me, but just wondering :)

Reply:
Oh, I see it is the latest version. Nevermind!